acceptance/tests/storeconfigs/collections_with_queries.rb

```diff
@@ -1,120 +0,0 @@
-test_name "collections with queries" do
```

Comment:
Same again, not sure why this is being removed? So the ticket was to remove the _legacy_ storeconfigs export tool, these tests are still relevant without the tool right? That is, we still have storeconfigs without the legacy export mechanism don't we?